hubspot/connection/__init__.py

```diff
@@ -204,6 +204,7 @@ def _require_successful_response(response):
             raise exception_class(
                 error_data['message'],
                 error_data['requestId'],
+                error_data,
```

Comment:
Hi @frankoid, about this, could you give me some example output for the failureMessages or a way for me to replicate them, just so I can see the structure for it, I'm thinking that if we have a proper structure, we could use a Record to store it instead of just a list straight out of their response in order to have a higher level of abstraction.

I'll look at their Documentation in the interim.